eng/targets/Npm.Common.targets

```diff
@@ -81,4 +82,30 @@
     </ItemGroup>
   </Target>
 
+
+  <Target Name="RunBrowserTests">
+    <Message Text="Running JavaScript tests" Importance="high" />
+
+    <!-- Skip the "inner" test run when we're running DailyTests -->
+    <Exec Command="npm run test:inner -- --no-color --configuration $(Configuration)"
+          Condition="'$(DailyTests)' != 'true'"
+          WorkingDirectory="$(RepositoryRoot)clients/ts/FunctionalTests"
+          IgnoreStandardErrorWarningFormat="true" />
+
+    <!-- Optionally run "daily test" run in Sauce Labs -->
+    <Error Text="Required property 'SauceUser' is missing!" Condition="'$(DailyTests)' == 'true' And '$(SauceUser)' == ''" />
+    <Error Text="Required property 'SauceKey' is missing!" Condition="'$(DailyTests)' == 'true' And '$(SauceKey)' == ''" />
+
+    <PropertyGroup>
+      <BrowserTestHostName Condition="'$(CI)' == 'true'">sauce.local</BrowserTestHostName>
```

Comment:
Move all this logic to SignalR.Npm.FunctionalTests.npmproj and just set `<NpmTestArgs>` to the sauce arguments when `$(DailyTests) == 'true'`